jobs/e2e_node/perf-config.yaml

```diff
@@ -0,0 +1,9 @@
+---
+images:
```

Comment:
Can we name this file as 
perf-image-config.yaml? Which is more explicit.

Reply:
yes! will rename.

Reply:
I agree it's more explicit, but there's precedent for `bleh-config` such as `benchmark-config.yaml`.

I think it would be great to have `image` in the name.

Reply:
Done.